lib/stat_tracker.rb

```diff
@@ -56,3 +56,116 @@ def generate_seasons
   # end
 
 end
+
+#########HOW I ORIGINALLY SET UP STATTRACKER AND WINNINGEST_COACH############
```

Comment:
Thanks for this work Drew! Ended up being able to cut the time down significantly